.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+  push:
+    branches: [release]
+  workflow_dispatch:
+    branches: [release]
+  workflow_call:
+    inputs:
+      QT_DEPENDENCIES:
+        description: 'Set to true to install Qt dependencies (Linux)'
+        required: false
+        default: 'false'
+        type: string
+
+jobs:
+  run-tests:
+    uses: nxt-dev/nxt/.github/workflows/unittests.yml@release
+    with:
+      QT_DEPENDENCIES: ${{ inputs.QT_DEPENDENCIES }}
+  build-and-deploy-pypi:
+    needs: [run-tests]
+    runs-on: ubuntu-latest
+    env:
+      TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Checkout build graph (if needed)
+        if: github.repository != 'nxt-dev/nxt'
+        uses: actions/checkout@v4
+        with:
+          repository: nxt-dev/nxt
+          ref: release
+          path: nxt
+          sparse-checkout: build
+
+
+      - name: Install Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Ensure PIP
+        run: python -m ensurepip --upgrade
+
+      - name: Install nxt-core (if needed)
+        if: github.repository != 'nxt-dev/nxt'
+        run: |
+          pip install git+https://github.com/nxt-dev/nxt.git@release
+
+      - name: Install this package and dependencies
+        run: |
+          pip install twine
+          pip install .
+
+      - name: Build package and upload
+        env:
+          # Packaging graph expects nxt to be cloned next to nxt_editor
+          # We hack the file fall backs to trick it into working
+          NXT_FILE_ROOTS: '../docs/api_docs'
```

Comment:
The env var is only needed by the editor right? Is it easy to move just this value there? if not oh well.

Reply:
Pushed a fix and it seems to [work on the test repo](https://github.com/nxt-dev/nxt_editor_test/actions/runs/9438359215/job/25995256986).